src/pkgmt/dev.py

```diff
@@ -43,7 +43,7 @@ def setup(c, version=None, doc=False):
     if not shutil.which("conda"):
         raise CommandError("conda not installed. Install it an try again.")
 
-    cfg = PyprojectConfig().get_config()
+    cfg = Config().from_file("pyproject.toml")
```

Comment:
```suggestion
    cfg = Config.from_file("pyproject.toml")
```

this is a class method

Reply:
fixed